pkg/fission-cli/cmd/package/update.go

```diff
@@ -175,23 +175,23 @@ func UpdatePackage(input cli.Input, client client.Interface, pkg *fv1.Package) (
 		}
 	}
 
-	newPkgMeta, err := client.V1().Package().Update(pkg)
+	newPkgMeta, err := client.FissionClientSet.CoreV1().Packages(pkg.ObjectMeta.Namespace).Update(input.Context(), pkg, metav1.UpdateOptions{})
 	if err != nil {
 		return nil, errors.Wrap(err, "update package")
 	}
 
 	fmt.Printf("Package '%v' updated\n", newPkgMeta.GetName())
 
-	return newPkgMeta, err
+	return &newPkgMeta.ObjectMeta, err
 }
 
-func UpdateFunctionPackageResourceVersion(client client.Interface, pkgMeta *metav1.ObjectMeta, fnList ...fv1.Function) error {
+func UpdateFunctionPackageResourceVersion(context context.Context, client cmd.Client, pkgMeta *metav1.ObjectMeta, fnList ...fv1.Function) error {
```

Comment:
Please don't name variable as context use `ctx context.Context` instead. As package name is also context.